gg-pac-team3-back/src/controllers/adminController.js

```diff
@@ -0,0 +1,47 @@
+// Import the Firebase admin module
+const admin = require('../config/firebase');
+
+// Initialize Firestore
+const db = admin.firestore();
+
+// Middleware to verify teacher user
+const verifyTeacher = async (req, res, next) => {
+  const authHeader = req.headers.authorization;
+
+  // Check if the authorization header is present
```

Comment:
Hi Matthew, I've created detailed instructions for testing routes in Postman. I've made a document for it since it's quite long. Here's the link: https://drive.google.com/file/d/1NN4oJBt2Vz3mf63mI32VRY0c5DN7mEUp/view?usp=drive_link. If you prefer a ready-made Postman Collection, I will also send an invite to our Postman workspace so you don't have to type every single route.